common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -739,6 +787,21 @@ def get_transcripts_info(self, is_bumper=False):
             language_code: transcript_file
             for language_code, transcript_file in transcripts.items() if transcript_file != ''
         }
+
+        # For phase 2, removing `include_val_transcripts` will make edx-val
+        # taking over the control for transcripts.
+        if include_val_transcripts:
+            transcript_languages = get_available_transcript_languages(
+                edx_video_id=self.edx_video_id,
+                youtube_id_1_0=self.youtube_id_1_0,
+                html5_sources=self.html5_sources
+            )
+            for language_code in transcript_languages:
+                if language_code == 'en' and not sub:
+                    sub = 'dummy_val_en_that_wont_be_in_contentstore'
+                elif not transcripts.get(language_code):
+                    transcripts[language_code] = 'dummy_val_that_wont_be_in_contentstore'
```

Comment:
Either use actual codes to clean this out OR make sure that we don't hit contentstore with these dummy values.